javaslang/src/test/java/javaslang/collection/TreeMapTest.java

```diff
@@ -37,7 +37,7 @@ protected String className() {
 
     @Override
     protected <T1 extends Comparable<? super T1>, T2> TreeMap<T1, T2> emptyMap() {
-        return TreeMap.empty(nullsFirst());
```

Comment:
Using nullsFirst() was terribly wrong. It made unit tests artificially work which were not intended to work.